packages/app-mobile/utils/ShareUtils.test.ts

```diff
@@ -0,0 +1,32 @@
+import { describe, test, expect } from '@jest/globals';
+import { mkdtempSync, rmdirSync } from 'fs';
+import { tmpdir } from 'os';
+import { join } from 'path';
+import shim from '@joplin/lib/shim';
```

Comment:
For tests, please use `fs` or `fs-extra` since they'll only run in Node

Reply:
Changed!